lib/logstash/filters/geoip.rb

```diff
@@ -88,6 +86,12 @@ class LogStash::Filters::GeoIP < LogStash::Filters::Base
   # is still valid GeoJSON.
   config :target, :validate => :string, :default => 'geoip'
 
+  # Specify the database type. By default "city" is selected (This is the type of the built-in GeoLite2 City database)
+  config :dbtype, :validate => ['country', 'city', 'anonymousIp', 'connectionType', 'domain', 'enterprise', 'isp'  ], :default => 'city'
```

Comment:
Since the database file seems to infer the database type, can we avoid this setting and use the database file to determine what the database type is?

This inference is what the geoip plugin did before we upgraded to geoip2:

https://github.com/logstash-plugins/logstash-filter-geoip/blob/v2.0.0/lib/logstash/filters/geoip.rb#L104-L115